WalletWasabi.Fluent/Views/Wallets/Receive/ReceiveAddressesView.axaml

```diff
@@ -34,17 +34,17 @@
       </DataGrid.Styles>
 
       <DataGrid.Columns>
-        <DataGridTemplateColumn Header="Reference"
+        <DataGridTemplateColumn Header="Labels"
                                 Width="*"
                                 CanUserSort="True"
                                 SortMemberPath="Label">
```

Comment:
Change it to `Labels`, this will fix the exception when clicking on the header, although the sorting won't work... We might have to improve the `SmartLabel` class

Reply:
wit is labels?

Reply:
In the `AddressViewModel` the property is called `Labels` and the `SortMemberPath` must be the property name on which the sorting should be based.